src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {
+  forwardRef,
+  useRef,
+  useImperativeHandle,
+  ChangeEventHandler,
+  ChangeEvent,
+} from 'react';
+import styles from './Checkbox.module.scss';
+
+type CheckboxPropsT = {
+  label?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  classProp?: string;
+  labelClassProp?: string;
+  onChange: Function;
+};
+
+const Checkbox = forwardRef(
+  ({ classProp, label, disabled, checked, onChange }: CheckboxPropsT, ref) => {
+    const inputRef = useRef<HTMLInputElement>(null);
+
+    /**
+     * Exposed Component API
+     */
+    useImperativeHandle(ref, () => {
+      return {};
+    });
```

Comment:
Is this just standard boilerplate you're adding to all components? Looks like `inputRef` isn't actually used for anything.

Reply:
yea I can remove that, I'm anticipating the need for that though but I can wait till it comes up.

Reply:
Gotcha. No need to remove it if you do plan on using it. I was just confirming.